### PR TITLE
Add AT_SECURE auxiliary value to loader/elf.py

### PR DESCRIPTION
**Pull request by justfoxing**
Helps support Android native binaries (bionic libc will abort during init if AT_SECURE isn't present in the auxiliary vector).

At this stage, just sets 0 (non-secure) for the value.

Issue #231

### CODE REVIEW

**Comment by aquynh** [comment]
Nice work! Can you please add a testcase into examples/ as well?

**Comment by justfoxing** [comment]
Sure, I'll put something together.

**Comment by justfoxing** [comment]
Testcase added for Android on arm64 (arm32 is something I'm still trying to solve).

File: tests/test_android.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    except:
+        regreturn = -1
+
+    ql.nprint("getrandom(0x%x, 0x%x, 0x%x) = %d" %
+              (buf, buflen, flags, regreturn))
+
+    if data:
+        ql.dprint(0, "[+] getrandom() CONTENT:")
+        ql.dprint(0, str(data))
+    ql_definesyscall_return(ql, regreturn)
```
Comment (xwings) [inline tests/test_android.py:33]: hi, we did a change so you can call
```
ql.os.definesyscall_return
```

can you update and try it. thanks

Comment (justfoxing) [reply tests/test_android.py:33]: Hi, I'm not seeing that change in either dev or master branches yet (I'm getting "'QlOsLinux' object has no attribute 'definesyscall_return'" when I run). Has it been merged in yet?

Comment (justfoxing) [reply tests/test_android.py]: Haha, yes it has, I was just failing to see it, my bad. Changed over to ql.os.definesyscall_return.

**Comment by xwings** [comment]
perfect ! i will add it in to the test

**Pull request merged**